.eleventy.js

```diff
@@ -103,6 +103,19 @@ module.exports = function(config) {
     return feedbackFormName;
   })
 
+  config.addNunjucksShortcode('readingTime', (text) => {
+    let textLength = text.split(' ').length
+    const wordsPerMinute = 150
+    const value = Math.ceil(textLength / wordsPerMinute)
+    if(value > 15){
+      return `больше 15 мин`
+    } else if (value < 5) {
+      return `меньше 5 мин`
+    } else {
+      return `${value} мин`
```

Comment:
```suggestion
      return `около ${value} мин`
```

Может стоит добавить больше неопределённости?

Reply:
Зачем?

Reply:
Скорость чтения у людей разная, но это не обязательный коммент. Можно зарезолвить.

Reply:
Так мы же не скорость отмеряем, а задом ориентир. Каждый понимает, какова скорость чтения относительно нашей оценки